jekyll/_cci2/api_v2_introduction.md

```diff
@@ -0,0 +1,226 @@
+
+More importantly, CircleCI found that more teams are using CircleCI for a much broader array of automated processes than the word "build" implies. “Pipelines” more accurately reflects the broad applications of our platform, therefore, "pipelines" are more appropriate for the set of stages CircleCI performs in processing your automation instructions against changes in your code. 
+
+#### Will I Have to Use Pipelines?
+
+You are not required to use pipelines, but note that pipelines will eventually be enabled for all projects. This will allow CircleCI to provide more powerful configuration and automation, as well as better optimization of your resources and more reliable performance. The overarching goal of this approach is for the transition to pipelines to be seamless and have no disruptions on any of your existing workflows.
+
+### Pipeline Parameters
+
+Pipeline parameters are new with API v2. To use pipeline parameters you must use configuration version 2.1 or higher.
+
+#### Declaring and Using Pipeline Parameters in a Configuration
+
+Pipeline parameters are declared using a parameters stanza in the top level keys of your `.circleci/config.yml` file. You may then reference the value of the parameter as a config variable in the scope `pipeline.parameters`.
+
+The example belows shows a configuration with two pipeline parameters, `image-tag` and `workingdir` both used on the subsequent config stanzas:
+
+```yaml
+version: 2.1
+
+parameters:
+  image-tag:
+    type: string
+    default: "latest"
+  workingdir:
+    type: string
+    default: "~/main"
+
+jobs:
+  build:
+    docker:
+      - image: circleci/node:<< pipeline.parameters.image-tag >>
+    environment:
+      IMAGETAG: << pipeline.parameters.image-tag >>
+    working_directory: << pipeline.parameters.workingdir >>
+    steps:
+      - run: echo "Image tag used was ${IMAGETAG}"
+      - run: echo "$(pwd) == << pipeline.parameters.workingdir >>"
+```
+
+#### Passing Parameters When Triggering Pipelines Via the API
+
+You may use the API v2 endpoint to trigger a pipeline, passing the parameters key in the JSON packet in your `POST` body.
+
+**NOTE:** To pass a parameter when triggering a pipeline via the API, the parameter must be declared in the configuration file.
+
+```
+curl -u ${CIRCLECI_TOKEN}: -X POST --header "Content-Type: application/json" -d '{
+  "parameters": {
+    "workingdir": "./myspecialdir",
+    "image-tag": "4.8.2"
+  }
+}' https://circleci.com/api/v2/project/:vcs-type/:username/:project/pipeline
```

Comment:
is "username" the way we talk about the org name?

Reply:
Actually, let's please use :project_slug here

Reply:
Fixed